### DISCUSSION TITLE
How to train token?

### DISCUSSION THREAD

**Discussion by LGH1gh** [facebookresearch/esm#146] [Q&A]
Hello everyone,

Thanks for your amazing job. We are currently trying to modify the pretrain objective on ESM-1v. There are 2 question coming up during implementation:

1) In MSA-1b paper, you used "masked language modeling pretraining task" without NSP objective. In the esm-1b alphabet, there are tokens such as <cls>, <eos>, and <unk>. I'm curious how did you use them in pre-train task?

2) In BERT, input sequences are shaped like '[cls] sequence-1 [seq] sequence-2 [seq]'. Did you use such format or others?

Thanks.
